habitat-hitl/habitat_hitl/_internal/networking/networking_process.py

```diff
@@ -231,18 +238,22 @@ async def send_keyframes(self) -> None:
             inc_keyframes_and_messages = (
                 self._interprocess_record.get_queued_keyframes()
             )
-            inc_keyframes = self._interprocess_record.get_queued_keyframes()
 
             if len(inc_keyframes_and_messages) > 0:
-                # consolidate all inc keyframes into one inc_keyframe
+                # Consolidate all inc keyframes into one inc_keyframe
                 tmp_con_keyframe = inc_keyframes_and_messages[0]
                 if len(inc_keyframes_and_messages) > 1:
```

Comment:
Seems odd to be discarding these messages only for the case of more than one keyframe.